_shared/project/db.py

```diff
@@ -0,0 +1,62 @@
+import alembic.command
+import alembic.config
+{% if cookiecutter.get("_directory") == "pyramid-app" %}
+import zope.sqlalchemy
+{% endif %}
+from sqlalchemy import MetaData{% if cookiecutter.get("_directory") == "pyramid-app" %}, create_engine{% endif %}, text
+from sqlalchemy.exc import ProgrammingError
+from sqlalchemy.orm import DeclarativeBase, MappedAsDataclass{% if cookiecutter.get("_directory") == "pyramid-app" %}, Session{% endif %}
+
+
+
+class Base(MappedAsDataclass, DeclarativeBase):
```

Comment:
`MappedAsDataclass` enables [SQLAlchemy's dataclasses support](https://docs.sqlalchemy.org/en/20/orm/dataclasses.html) which is awesome, for example your model classes get good `__repr__()`'s, you can use `dataclasses.asdict()` and `dataclasses.astuple()` on them, etc.